frontend/src/v5/ui/routes/viewer/drawings/drawingCalibrationMenu/drawingCalibrationMenu.component.tsx

```diff
@@ -35,10 +36,16 @@ type DrawingsCalibrationMenuProps = DashboardListItemButtonProps & {
 export const DrawingsCalibrationMenu = ({ calibrationState, onCalibrateClick, drawingId, disabled, ...props }: DrawingsCalibrationMenuProps) => {
 	const { teamspace, project } = useParams<DashboardParams>();
 	const disableButton = disabled || calibrationState === CalibrationState.EMPTY;
+	const latestRevision = DrawingRevisionsHooksSelectors.selectLatestActiveRevision(drawingId);
 
-	const approveCalibration = () => DrawingsActionsDispatchers.updateDrawing(teamspace, project, drawingId, {
-		calibration: CalibrationState.CALIBRATED,
-	});
+	const onApproveCalibration = () => {
```

Comment:
The calibration vectors are not appearing in one case:
- I upload a new revision in the drawing list.
- I click on calibrate in the ellipsis menu
- finish calibration
- click this time on recalibrate 

what happens:
- the calibration vectors dont appear
-
what should happen:
- the previous calibration should appear